modules/custom/social_react/social_react.libraries.yml

```diff
@@ -0,0 +1,27 @@
+react:
+  remote: https://github.com/facebook/react
+  version: 16.7
+  license:
+    name: MIT
+    url: https://github.com/facebook/react/blob/master/LICENSE
+    gpl-compatible: true
+  js:
+    https://unpkg.com/react@16.7/umd/react.production.min.js:
```

Comment:
Mostly just following the React documentation and not thinking about being able to download NPM packages with composer 🙂 I'll change it

Reply:
Discussed with Ronald and he's convinced me to go with Asset Packagist. Big reason for this was some issues in Drupal core that can cause race conditions when loading from a CDN instead of packaging everything inline using Drupal's Javascript compilation.

Although I suspect we won't run into the race condition here, moving back to unpkg is a relatively easy change 🙂